sql/main_summary_v5.sql

```diff
@@ -0,0 +1,357 @@
+
+--
+SELECT
+  document_id,
+  client_id,
+  sample_id,
+  metadata.uri.app_update_channel AS channel,
+  normalized_channel,
+  normalized_os_version,
+  metadata.geo.country,
+  metadata.geo.city,
+  metadata.geo.subdivision1 AS geo_subdivision1,
+  metadata.geo.subdivision2 AS geo_subdivision2,
+  environment.system.os.name AS os,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.environment.system.os.version") AS os_version,
+  SAFE_CAST(environment.system.os.service_pack_major AS INT64) AS os_service_pack_major,
+  SAFE_CAST(environment.system.os.service_pack_minor AS INT64) AS os_service_pack_minor,
+  SAFE_CAST(environment.system.os.windows_build_number AS INT64) AS windows_build_number,
+  SAFE_CAST(environment.system.os.windows_ubr AS INT64) AS windows_ubr,
+
+  -- Note: Windows only!
+  SAFE_CAST(environment.system.os.install_year AS INT64) AS install_year,
+  environment.system.is_wow64,
+
+  SAFE_CAST(environment.system.memory_mb AS INT64) AS memory_mb,
+
+  environment.system.cpu.count AS cpu_count,
+  environment.system.cpu.cores AS cpu_cores,
+  environment.system.cpu.vendor AS cpu_vendor,
+  environment.system.cpu.family AS cpu_family,
+  environment.system.cpu.model AS cpu_model,
+  environment.system.cpu.stepping AS cpu_stepping,
+  SAFE_CAST(environment.system.cpu.l2cache_kb AS INT64) AS cpu_l2_cache_kb,
+  SAFE_CAST(environment.system.cpu.l3cache_kb AS INT64) AS cpu_l3_cache_kb,
+  SAFE_CAST(environment.system.cpu.speed_m_hz AS INT64) AS cpu_speed_mhz,
+
+  environment.system.gfx.features.d3d11.status AS gfx_features_d3d11_status,
+  environment.system.gfx.features.d2d.status AS gfx_features_d2d_status,
+  environment.system.gfx.features.gpu_process.status AS gfx_features_gpu_process_status,
+  environment.system.gfx.features.advanced_layers.status AS gfx_features_advanced_layers_status,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.environment.system.gfx.features.wr_qualified.status") AS gfx_features_wrqualified_status,
+  JSON_EXTRACT_SCALAR(additional_properties, "$.environment.system.gfx.features.webrender.status") AS gfx_features_webrender_status,
```

Comment:
These two fields are missing from the [Firefox source docs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/data/environment.html).